projects/declaration-files/global-travels/01-passport-declarations/README.md

```diff
@@ -2,9 +2,9 @@
 
 Ah, thank you for volunteering.
 Since you're on an international flight, we will need to validate your passport before you can leave the designated travel area.
-But: our systems are quite legacy and we've lost the type annotations for our passport checking system.
+But our systems are quite legacy and we've lost the type annotations for our passport checking system.
```

Comment:
This one technically isn't a typo (the `:` is intentional) but on re-read I like this updated version more. 😄